common/thrift_client_pool.h

```diff
@@ -167,6 +175,10 @@ class ThriftClientPool {
         if (FLAGS_channel_send_timeout_ms > 0) {
           channel->setTimeout(FLAGS_channel_send_timeout_ms);
         }
+        if (binary_protocol) {
```

Comment:
We can use USE_BINARY_PROTOCOL here directly. And don't need to change the interface for EventLoop.